spec/middleware/authentication_spec.rb

```diff
@@ -16,6 +16,15 @@
 
         expect(status).to eq(200)
       end
+
+      it 'allows the request if given a correct password header and a Passwords header' do
+        app = lambda { |env| [200, {'Content-Type' => 'application/json'}, Array.new] }
+        middleware = Napa::Middleware::Authentication.new(app)
+        env = Rack::MockRequest.env_for('/test', {'HTTP_PASSWORD' => 'foo', 'HTTP_PASSWORDS' => 'foo,bar'})
+        status, headers, body = middleware.call(env)
```

Comment:
Useless assignment to variable - `headers`. Use `_` or `_headers` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `body`. Use `_` or `_body` as a variable name to indicate that it won't be used.